src/tasks/installers/operator.ts

```diff
@@ -162,6 +162,40 @@ export class OperatorTasks {
           }
         }
       },
+      {
+        title: 'Configure host pvc and storage class',
+        enabled: () => {
+          if (flags['pvc-host-volume-path']) {
```

Comment:
AFAIU this flag is ignored if there is already deployed Che that was stopped before. Maybe we should print a warning that user must execute `server:delete` and then `server:start` if he wants they configuration parameters to be involved?
I saw something similar when you're trying to tune memory limit for existing `crc` instance.
It may be actual for existing flags as well, so I'm OK to handle it as a separate issue.